lightly/transforms/moco_transform.py

```diff
@@ -93,4 +93,55 @@ def __init__(
         )
 
 
-MoCoV2Transform = SimCLRTransform  # MoCo v2 uses the same transform as SimCLR
+class MoCoV2Transform(SimCLRTransform):
```

Comment:
I am not sure whether this is necessary. However, your call to add the duplicate code.

Reply:
It is necessary to render the docstring in the docs. There is no code duplicated. It is not ideal but not horrible either.